Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -228,7 +228,7 @@ class LoginsHelper: TabHelper {
             }
 
             let json = JSON(jsonObj)
-            let src = "window.__firefox__.logins.inject(\(json.rawString()))"
+            let src = "window.__firefox__.logins.inject(\(json.rawString()!))"
```

Comment:
Hmm.. I don't know if it's possible since jsonObj is a somewhat hard-coded dictionary above. But I suppose it's possible? I can throw together a follow-up patch.